mir_eval/display.py

```diff
@@ -12,7 +12,7 @@
 from matplotlib.ticker import FuncFormatter, MultipleLocator
 from matplotlib.ticker import Formatter
 from matplotlib.colors import LinearSegmentedColormap, LogNorm, ColorConverter
-from matplotlib.collections import BrokenBarHCollection
+from matplotlib.collections import PolyCollection as BrokenBarHCollection
```

Comment:
We can actually just remove this import entirely.  It's no longer used.

Reply:
Awesome! Thanks for pointing this out.